src/runtime/test/particle-execution-context-test.ts

```diff
@@ -34,14 +34,14 @@ describe('Particle Execution Context', () => {
           };
         });`
       }),
-      slotComposer: new FakeSlotComposer(),
+      slotComposer: new MockSlotComposer({strict:false}).newExpectations('debug'),
```

Comment:
Why are you replacing FakeSlotComposer with MockSlotComposer? In general, MockSlotComposer is needed when testing rendering expectations. For all other tests, the Fake one should be fine.

Reply:
if for some reason we do want to use MockSlotComposer in all tests, maybe add a static helper method to create a `new MockSlotComposer({strict: false}).newExpectations('debug')`?

Reply:
The slotComposer for TestHelper requires this as part of it's arguments.  When these fakes/mocks were in javascript they were just treated as 'any', so they did not generate problems.

Should we change TestHelper instead?  The validation login in MockSlotComposer looks like it could be useful for these tests..  (And now that everything is in typescript I could replace with a proper mocking solution..)

Reply:
Yes, i think we could change TestHelper to allow for any SlotComposer (and/or whatever extends it) - i think TestHelper simply predated FakeSlotComposer, hence the limitation.